packages/rsu_client.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Rsu_client < Package
+  description 'RSU-Client is a RuneScape Client Launcher written for the now Legacy client and now OldSchool.'
+  homepage 'https://github.com/rsu-client/rsu-client'
+  version '4.3.7'
+  source_url 'https://github.com/rsu-client/rsu-client/archive/v4.3.7.tar.gz'
+  source_sha256 '7cc589de111baad956c0c41a4d7f1b45168886bf623dd3886d07fa3d1dfd604c'
+
+  depends_on 'jdk8'
+  depends_on 'p7zip'
+  depends_on 'wxwidgets'
+  depends_on 'xdg_utils'
+  depends_on 'sommelier'
+
+  def self.patch
+    Dir.chdir 'runescape' do
+      system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
+                sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' $f
+              done"
+      system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
+                sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' $f
+              done"
+    end
+  end
+
+  def self.build
+    system 'cpan install Archive::Extract exit'
+  end
+
+  def self.install
+    system 'mkdir', '-pv', "#{CREW_DEST_PREFIX}/share"
```

Comment:
Could you please remove the verbosity in this package?
`ln -sv` > `ln -s`, `w /dev/stdout` > ``, `cp -rv` > `cp -a`, and `mkdir -pv` > `mkdir -p`
`cp -a` preserves permissions.

Reply:
Yes, I was going to address that.